hack/verify-examples-kind.sh

```diff
@@ -105,9 +116,6 @@ for CHANNEL in experimental standard; do
         echo Examples failed as expected
 done
 
-# Undo workaround from earlier
-sed -i 's/latest/v0.5.1/g' config/webhook/admission_webhook.yaml
```

Comment:
Removing this line means that running this command locally will leave `config/webhook/kustomization.yaml` in your git diff. Do you mind adding `rm config/webhook/kustomization.yaml` here?

Reply:
OK.

Reply:
OK

Reply:
I think leaving the remove logic in `cleanup` would be better.